zrml/prediction-markets/src/benchmarks.rs

```diff
@@ -487,6 +487,38 @@ benchmarks! {
         assert_eq!(pool.pool_status, PoolStatus::Active);
     }
 
+    start_global_dispute {
+        // CacheSize::get() does not work
+        let m in 1..62;
```

Comment:
69a206f394ec63025b5195af7047c7d20db280ff

Reply:
Aren't we good, when the benchmark tests don't fail after fixing it https://github.com/zeitgeistpm/zeitgeist/commit/69a206f394ec63025b5195af7047c7d20db280ff? 

Oh no I didn't, but I try to reproduce it: 

I mean the errors depend on each other. Otherwise the tests should still fail after fixing it. I notice the order of the test benchmark failures is the same as in the benchmark.rs file. So I assume this is an internal benchmark problem. The only benchmark which actually failed was `start_global_disputes` at this time (which is also the first in the order).

I don't exactly know why this happens, but I have a good assumption. Something like this called twice could lead to this kind of error. So I assume, that frame_benchmarking has some internal bug, which executes the all benchmark tests, but the storage interferes with it (when one benchmark produces an error).

```rust
T::AssetManager::deposit(Asset::Ztg, &disputor, (u128::MAX).saturated_into())?;
```

Reply:
Thanks for looking into this. Yeah, it seems like a good guess to assume that one of the tests breaks state and then other tests fail. Still, very concerning. I'll try to reproduce this in a minimal example.